core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -194,6 +199,21 @@ var ExplorationEditorTranslationTab = function() {
     return nodeElement.element(by.css('.protractor-test-node-label'));
   };
 
+  var audioUploadInputElem = element(
+    by.css('.protractor-test-upload-audio'));
+  var audioOverFiveMinutesErrorMessageElement = element(
```

Comment:
let's be consistent... Let's either name the variables as elem or element.

Reply:
Done